ginkgo.Jenkinsfile

```diff
@@ -36,55 +36,27 @@ pipeline {
                 sh '/usr/local/bin/cleanup || true'
             }
         }
-        stage('Precheck') {
+        stage('Prechecks+unittesting') {
```

Comment:
Add a comment to the Jenkinsfile explaining why these stagings are combined, as it's not immediately obvious.